summingbird-online/src/test/scala/com/twitter/summingbird/online/PlannerSpec.scala

```diff
@@ -62,6 +63,75 @@ class PlannerSpec extends WordSpec {
   def arbSource1 = sample[Producer[Memory, Int]]
   def arbSource2 = sample[KeyedProducer[Memory, Int, Int]]
 
+  /*
+ * Test Case : A simple src.map.summer topology with FMMergeableWithSource opt-in functionality[ No FlatMappedProducer ].
+ * Asserts : Online Plan takes options as an argument
+ *           The plan considers the options to convert into a two node toplogy removing FlatMapNode.
+ */
+  "Basic Online Plan without a flat map and opt-in FMMergeableWithSource" in {
+    val store1 = testStore
+    val srcName = "first"
+    val fmName = "mapped"
+    val smName = "summer"
+    val h1 = arbSource1.name("first")
+      .map { i:Int => (i,i * i) }.name("mapped")
+      .sumByKey(store1)
+
+    val opts =  Map(
+      fmName  -> Options().set(SummerParallelism(10)).set(FlatMapParallelism(50)).set(SourceParallelism(50)).set(FMMergeableWithSource(true)),
```

Comment:
Do we want to have assertions around what happens with these parallelism settings?